File: examples/happybday.py
j=int(input("Input the number of Layers"))
n=j-1
y=n
u=1
i=0
print("\n \n")
while i<j:
    i+=1
    for z in range (n):
    	print('  ',end="")
    for q in range (u):
        print('||',end="")
    print ("")
    u+=2
    n-=1
 
for k in range (y):
    print ('  ',end="")
print ("")
print ("HAPPY BIRTHDAY")

File: examples/HappyBDay.java
import java.time.LocalDate;
import java.time.Month;

public class HappyBDay {
   public static void main(String args[]) {

    // declare variables for birthday
    int birthDate = 23;
    Month birthMonth = Month.SEPTEMBER;

    // get current date
    LocalDate currentDate = LocalDate.now();
    System.out.println("Todays Date: " + currentDate);

    // get current date and month
    int date = currentDate.getDayOfMonth();
    Month month = currentDate.getMonth();

    if(date == birthDate && month == birthMonth) {
      System.out.println("HAPPY BIRTHDAY TO YOU !!");
    }
    else {
      System.out.println("Today is not my birthday.");
    }
   }